posts/2023-10-11-introducing-grails-6_1_0.md

```diff
@@ -0,0 +1,142 @@
+author: Puneet Behl
+image: grails-blog-index-3.png
+---
+
+# [%title]
+
+[%author]
+
+[%date]
+
+The Grails Foundation is excited to announce the release of Grails Framework 6.1.0! This release comes with a host of changes, bug-fixes, and improvements that will further enhance your experience with the Grails framework . Grails continues to be a powerful and efficient framework for building web applications in Groovy, and this latest version takes it to the next level. Let's dive into what's new in Grails 6.1.0.
+
+## What's Changed
+
+In this release, we have made several changes to improve the framework's performance and maintainability. Some of the notable changes include:
+
+* **Update to SnakeYaml 2.2**: This update fixes the [CVE-2022-1471](https://www.cve.org/CVERecord?id=CVE-2022-1471) reported in December of last year. This unsafe deserialization problem could easily lead to arbitrary code execution under the right circumstances. For more details on this vulnerability, you can read about it on [Snyk's blog](https://snyk.io/blog/snakeyaml-unsafe-deserialization-vulnerability/).
+
+* **Decoupling the Sitemesh Plugin**: We have introduced significant changes to decouple the Sitemesh plugin for more flexibility. Here are the details:
+    1. Created a new `org.grails.plugins:sitemesh2` and moved Sitemesh specific tag libraries that require the Sitemesh2 dependency into a separate Sitemesh2 specific plugin. This plugin can now be optionally used, providing more flexibility in choosing Sitemesh versions.
+
+    2. Introducing the [Sitemesh3 Grails Plugin](https://github.com/codeconsole/grails-sitemesh3), a new plugin that can be independently used with Grails 6. This plugin offers compatibility with the latest Sitemesh version, allowing developers to leverage the benefits of Sitemesh3 in their Grails applications.
+
+    3. To migrate from Sitemesh 2 to Sitemesh 3, not only completely decouples Sitemesh from GSP, but it also introduces a powerful new feature that GSP has been lacking: Decorator Chaining. With SiteMesh 3, you can apply multiple decorators to a view. For instance, something like this:
+
+       ````html
+       <meta name="layout" content="my-decorator,my-other-decorator" />
+       ````
+
+       This enhancement provides more flexibility in structuring views and layouts, making it easier to create complex and dynamic page layouts.
+
+    4. Going Further (JDK 17) - Jakarta compatible SiteMesh 3.2 is already available. Sitemesh 3 supports JDK 17, making it an easier transition to future Grails releases, such as Grails 7.0.
+
+    5. To learn more about the discussions around Grails and Sitemesh, you can read the details at [here](https://github.com/grails/grails-core/issues/13058). This discussion provides insights into the decisions and considerations made during the decoupling process.
```

Comment:
"…you can read the details at [here](...link...)". I think a better pattern than linking "here" is to say something like "join the discussion" or "join the thread" and link that phrase. So something like:

> To learn more about updates to Grails and Sitemesh, please [review the discussion](https://github.com/grails/grails-core/issues/13058) which provides insights into the decisions and…

But that's personal preference.

Reply:
This make sense, can you please update the PR.

Reply:
Updated. I also renamed the file prefix to `2023-11-10-` to follow a YYYY-MM-DD format.